sst/test/include/framework/macros/util.hh

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#define CONCAT(A, B) A##B
+#define STRINGIFY(STR) #STR
```

Comment:
For the sake of consistency, Could be useful to have comments on these macro definitions to describe what they are doing

Reply:
Please take a look.